app/views/staff/shared/activities/_activity.html.haml

```diff
@@ -320,7 +320,7 @@
       - unless activity_presenter.fstc_applies.to_s.blank?
         = t("summary.label.activity.fstc_applies.#{activity_presenter.fstc_applies}")
     %dd.govuk-summary-list__actions
-      - if policy(activity_presenter).update? && step_is_complete_or_next?(activity: activity_presenter, step: :fstc_applies)
+      - if policy(activity_presenter).update? && step_is_complete_or_next?(activity: activity_presenter, step: :fstc_applies) && !can_infer_fstc?(@activity.aid_type)
```

Comment:
I wonder if it might be better to have an small method in the helpers here? Something like `ftsc_not_inferred?`, just to make the intention of this line a bit clearer?